docs/docs/cmd/spo/group/group-member-add.md

```diff
@@ -33,26 +33,75 @@ For the `--userName` or `--email` options you can specify multiple values by sep
 
 ## Examples
 
-Add a user with name _Alex.Wilber@contoso.com_ to the SharePoint group with id _5_ available on the web _https://contoso.sharepoint.com/sites/SiteA_
+Add a user with the userName parameter to a SharePoint group with the groupId parameter
```

Comment:
I'm missing the new `userId` option in the Options section. Could you add it there?